src/enemies/mod.rs

```diff
@@ -48,13 +48,13 @@ fn spawn_slime(
     rapier_config: &RapierConfiguration,
     asset_server: &AssetServer,
 ) {
-    let collider_radius = 24.0 * 2.0 / rapier_config.scale / 2.0;
+    let collider_radius = 26.0 / rapier_config.scale / 2.0;
 
     commands
         .spawn_bundle(SpriteBundle {
-            texture: asset_server.load("slime.png"),
+            texture: asset_server.load("drone.png"),
             transform: Transform {
-                scale: Vec3::new(2.0, 2.0, 1.0),
+                //scale: Vec3::new(1.0, 1.0, 1.0),
```

Comment:
nitpick: should we just remove this comment?

Reply:
yes